airflow/example_dags/example_sla_dag.py

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import time
+from datetime import datetime, timedelta
+
+from airflow.decorators import dag, task
+
+"""Example DAG demonstrating SLA use in Tasks"""
+
+
+# [START howto_task_sla]
+def sla_callback(*args):
```

Comment:
WDYT about having the same function name in the SLA Concepts guide and this example DAG? I'm thinking the SLA Concepts guide is a detailed breakdown of SLAs and this example DAG pulls it all together at the end. Having the functions named the same seems like a great way to bring some continuity to the doc.

Reply:
The print function prints all the args, matching it correctly for the user to see. I am of the opinion that it won't be confusing. Is it?

Reply:
Not saying it's confusing at all, this is a great example. I was just thinking since the Concepts doc has "Examples of ``sla_miss_callback`` function signature:..." noted it might be good for users to see the same function in the context of this example DAG as well as the other components that are outlined in the doc. Again, no strong opinions here.